usaspending_api/awards/models.py

```diff
@@ -334,7 +334,7 @@ class TransactionContract(DataSourceTrackedModel):
     naics = models.TextField(blank=True, null=True, verbose_name="NAICS", help_text="Specified which industry the work for this transaction falls into. A 6-digit code")
     naics_description = models.TextField(blank=True, null=True, verbose_name="NAICS Description", help_text="A plain text description of the NAICS code")
     period_of_performance_potential_end_date = models.DateField(max_length=10, verbose_name="Period of Performance Potential End Date", null=True, help_text="The potential end date of the period of performance")
-    ordering_period_end_date = models.CharField(max_length=8, blank=True, null=True, help_text="The end date for the ordering period")
+    ordering_period_end_date = models.TextField(blank=True, null=True, help_text="The end date for the ordering period")  # If we aren't converting it to a date, do we even really want it?
```

Comment:
@catherinedevlin good question here re: date...it might be an oversight from way back that we never caught. I checked the field definition in the schema docs, and it sounds like a legitimate date:

    For procurement, the date on which, for the award referred to by the action 
    being reported, no additional orders referring to it may be placed. This date 
    applies only to procurement indefinite delivery vehicles (such as indefinite 
    delivery contracts or blanket purchase agreements). Administrative actions 
    related to this award may continue to occur after this date. The period of 
    performance end dates for procurement orders issued under the indefinite 
    delivery vehicle may extend beyond this date.

We don't surface this field on the website (at least in any of the filters), which may explain why we haven't had a problem. I'd vote for fixing this now, before we load in a bunch of historic data. Warning: I haven't looked at said historic data to know what kind of shape the data is in.